build/configs/qemu/tc_64k/defconfig

```diff
@@ -379,6 +405,10 @@ CONFIG_MQ_MAXMSGSIZE=32
 # CONFIG_SCHED_HPWORK is not set
 # CONFIG_SCHED_LPWORK is not set
 
+#
+# Binary Manager
```

Comment:
There is no binary manager in master branch. I think you get this from protection branch.
Please update this in master.

Reply:
modify